spring-cloud-starter-stream-sink-jdbc/src/main/java/org/springframework/cloud/stream/app/jdbc/sink/JdbcSinkConfiguration.java

```diff
@@ -63,7 +81,9 @@
  * @author Eric Bottard
  * @author Thomas Risberg
  * @author Robert St. John
+ * @author Oliver Flasch
  */
+@Configuration
```

Comment:
The `@EnableBinding` comes already with the `@Configuration`

Reply:
Should be fixed in 8d33bd5c35048c2385412e905ebfd96ff43c3b16 , thanks for the explanation!